### ISSUE TITLE
Duplicate/Transient JS associated to classes

### ISSUE THREAD

**Issue by kbrebel04** [soflyy/oxygen-bugs-and-features#1823]
**Describe the bug**
If you put custom JS in any sort of div and copy settings from the ID to the CLASS it creates a copy of the JS inside of the CLASS but NOT in the oxygen interface. So you end up with JS on the front end of your site that you can only remove by resetting the CLASS.

The biggest problem with this is, we have no way to see what JS is associated with what class because Oxygen associated JS with IDs, not classes. When you copy settings from an ID to a CLASS and JS is in those settings, it creates a ghost JS script that is then associated to the CLASS that we are unable to modify or erase unless the class is completely wiped. 

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
Yes: http://silly-osprey.w6.wpsandbox.pro/test/

**To Reproduce**
1. Create a new page
2. Create a section
3. add a class to the section
4. add JS to the section
5. Copy settings from the ID to the CLASS
6. Go to the front end and search for the jQuery you made and you will see 2 occurrences of it. One associated to the ID, one associated to the class
7. Delete the JQuery from either the CLASS or ID of the section. 
8. Visit the front end again to find there is an entry of your jquery with a trailing "_orginal" with no way to find or edit it in the Oxygen Builder

**Expected behavior**
With currently Oxygen functionality, the JS editor is associated with object IDs. When using the COPY/PASTE feature for classes and IDs Oxygen should IGNORE copying JS from an ID to a CLASS.

**What actually happens**
Oxygen copies the JS from the ID and applies it to the CLASS in a way that can not be edited by the user unless the class is wiped.

**Comment by AlbertMarashi** [comment]
Also had this bug. Bitch to resolve.

TLDR to fix:
- delete the entire class in manage > selectors > .class

**Comment by KittenCodes** [comment]
Fixed in v3.9 Beta 1.
